### DISCUSSION TITLE
Upgrade from 4.12.0-0.okd-2023-04-16-041331 to 4.13.0-0.okd-2023-10-28-065448

### DISCUSSION THREAD

**Discussion by dapieper** [okd-project/okd#1787] [General]
Hi,
I would like to upgrade directly from 4.12.0-0.okd-2023-04-16-041331 to 4.13.0-0.okd-2023-10-28-065448.
Unfortunately there is no direct route and you would have to install 3 versions in between:
https://amd64.origin.releases.ci.openshift.org/releasestream/4-stable/release/4.12.0-0.okd-2023-04-16-041331

It is possible to jump directly to the version without any problems, e.g. with:
`oc adm upgrade --force --allow-explicit-upgrade=true --to-image quay.io/openshift/okd@sha256:29e8ad18b5d011c3fd8275ec9aad5964e4dcfd0138d229f2917acfcbefde930c`

Or can someone adjust the upgrade path to make a direct upgrade possible?

Many thanks.

**Comment by vrutkovs** [comment]
>It is possible to jump directly to the version

yes, but since this path is not tested in CI we can't verify that any problems occur. Also, CI setup is different from yours, so we can't guarantee that it would be problem-free.

However, most of the problems would have been known by now, so it seems safe to upgrade (provided you have etcd backups)
